### ISSUE TITLE
Question!

### ISSUE THREAD

**Issue by Yunus1903** [TheHolyWaffle/TeamSpeak-3-Java-API#47]
I really want to use this API.
But i have some questions:

- What about Licence?
- ^^ do i have to credit you?
- Something else?

**Comment by gutzeit** [comment]
Haven't you seen the License?
https://github.com/TheHolyWaffle/TeamSpeak-3-Java-API/blob/master/LICENSE

**Comment by Yunus1903** [comment]
Yes i have but licences of all kind are unreadable,
they should make it better...

**Comment by Kakifrucht** [comment]
What is there not to understand?
They are legal documents, obviously they are not written for kindergardeners.

**Comment by Firedroide** [comment]
This license is less than 200 words and one of the most commonly used licenses out there.

ELI5 breakdown here: https://tldrlegal.com/license/mit-license
